dev/Deployment/DeploymentManager.cpp

```diff
@@ -193,23 +195,68 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
     CATCH_RETURN()
 
     // Deploys all of the packages carried by the specified framework.
-    HRESULT DeploymentManager::DeployPackages(const std::wstring& frameworkPackageFullName) try
+    HRESULT DeploymentManager::Deploy(const std::wstring& frameworkPackageFullName) try
+    {
+        RETURN_IF_FAILED(InstallLicenses(frameworkPackageFullName));
+        RETURN_IF_FAILED(DeployPackages(frameworkPackageFullName));
+        return S_OK;
+    }
+    CATCH_RETURN()
+
+    HRESULT DeploymentManager::InstallLicenses(const std::wstring& frameworkPackageFullName)
+    {
+        // Build path for licenses
+        auto licensePath{ std::filesystem::path(GetPackagePath(frameworkPackageFullName)) };
+        licensePath /= WINDOWSAPPRUNTIME_FRAMEWORK_PACKAGE_FOLDER;
+        auto licenseFilespec{ licensePath };
+        licenseFilespec /= L"*-license.xml";
```

Comment:
were you testing with the placeholder license, by changes? The final xml's are *_license.xml.

Reply:
Yes just wrong filename amongst the manual hackery. Fixing

Reply:
Also note the licenses are in the MSIX folder

Reply:
Yup

`#define WINDOWSAPPRUNTIME_FRAMEWORK_PACKAGE_FOLDER                  L"MSIX"`

IOW

```
licensePath = fwk pkgdir
licensePath += "\MSIX"
licensePath += "\*-license.xml"
```